lib/audit/catalog_to_moab.rb

```diff
@@ -142,4 +143,23 @@ def online_moab_found?(druid, storage_dir)
     false
   end
 
+  # given whether the caller found the expected version of preserved_copy on disk, this will perform
+  # other validations of what's on disk, and will update the status accordingly.
```

Comment:
since this method essentially duplicates a method in pohandler results, please add the comment implying that?  (see other private methods)